inspec.yml

```diff
@@ -8,48 +8,130 @@ version: 2.1.2
 inspec_version: ">= 4.0"
 
 inputs:
-  - name: disable_slow_controls
-    description: Controls that are known to consistently have long run times can be disabled with this attribute
-    type: Boolean
-    value: false
+  - name: org_name
+    desc: "Name of the organization running this profile"
+    type: Hash
+    value:
+      acronym: "DoD"
+      full_form: "Department of Defense"
 
-  #SV-230548
-  - name: container_host
-    description: Flag to designate if the target is a container host
-    type: Boolean
-    value: false
+#SV-230244, SV-244525:
+  - name: mins_of_inactivity
+    desc: Maximum number of minutes of account inactivity before account lockout
+    type: Numeric
+    value: 15
+
+  - name: system_inactivity_timeout # also used in SV-230353
+    desc: Maximum system inactivity timeout (time in seconds).
+    type: Numeric
+    value: 900
```

Comment:
I need to take a closer look at this input (which was previously used in SV-244525 but was replaced with `client_alive_interval` later). The only potential use of this input seems to be in SV-230353 which is already covered by `fail_interval`

Reply:
SV-230353 is a totally separate requirement, and had always used system_inactivity_timeout. I do not see SV-230353 ever using anything else.
https://github.com/CMSgov/redhat-enterprise-linux-8-stig-baseline/blob/fa06c0a620740e98549ffdd4089d2ef3ed7bc490/controls/SV-230353.rb#L52

SV-244525 has only ever used client_alive_interval:
https://github.com/CMSgov/redhat-enterprise-linux-8-stig-baseline/blob/fa06c0a620740e98549ffdd4089d2ef3ed7bc490/controls/SV-244525.rb#L79

Reply:
We're looking at two different control files (and I might be missing context/understanding for this), but looking at the parameterized version of [SV-230353](https://github.com/ssayed118/redhat-enterprise-linux-8-stig-baseline/blob/vagrant/controls/SV-230353.rb), `fail_interval` and `fail_interval_mins` are used and I'm just wondering why we are also using `system_inactivity_timeout` when `fail_interval` has the same value of 900.

If the overlays have different values for `system_inactivity_timeout` and `fail_interval`, then it's totally understandable!

Reply:
After our discussion, we have determined to use only the `system_inactivity_timeout` input in [SV-230353](https://github.com/ssayed118/redhat-enterprise-linux-8-stig-baseline/blob/vagrant/controls/SV-230353.rb) and create more inputs as necessary specifically related to `system_inactivity_timeout`

Reply:
https://github.com/ssayed118/redhat-enterprise-linux-8-stig-baseline/pull/1